rust/abacus-base/src/settings/chains.rs

```diff
@@ -103,6 +107,7 @@ impl ChainSetup<InboxAddresses> {
                             .into(),
                     },
                     signer,
+                    InboxArgs {},
```

Comment:
I see some of these types are suffixed with `Args` and others `Params`- I don't have a preference on which one, but I think consistency is important